resources/lang/da.json

```diff
@@ -0,0 +1,345 @@
+	"Are you sure you want to run this action?": "Er du sikker på du vil køre denne handling?",
+	"Attach": "Vedhæft",
+	"Attach &amp; Attach Another": "Vedhæft &amp; vedhæft en anden",
+	"Cancel": "Annullere",
+	"Choose": "Vælg",
+	"Choose File": "Vælg fil",
+	"Choose Type": "Vælg type",
+	"Choose an option": "Vælg en indstilling",
+	"Click to choose": "Klik for at vælge",
+	"Create": "Opret",
+	"Create &amp; Add Another": "Opret &amp; tilføj en anden",
+	"Delete File": "Slet fil",
+	"Edit": "Rediger",
+	"Edit Attached": "Rediger Vedhæftet",
+	"Go Home": "Til forside",
+	"Hold Up!": "Vent et øjeblik!",
+	"Lens": "Linse",
+	"New": "Ny",
+	"Next": "Næste",
+	"Only Trashed": "Kun slettede",
+	"Per Page:": "Per side:",
+	"Please select a resource to perform this action on.": "Vælg venligst en ressource at udfører denne handling på.",
+	"Preview": "Forhåndsvisning",
+	"Previous": "Forrige",
+	"Run Action": "Kør handling",
+	"Select Action": "Vælg handling",
+	"Select All": "Vælg alle",
+	"Select All Matching": "Vælg alle matchende",
+	"Something went wrong.": "Noget gik galt.",
+	"The action ran successfully!": "Handlingen kørte med succes!",
+	"The government won\\'t let us show you what\\'s behind these doors": "Regeringen vil ikke lade os vise dig, hvad der er bag disse døre",
+	"This image": "Dette billede",
+	"Update": "Opdater",
+	"Update &amp; Continue Editing": "Opdater &amp; forsæt redigering",
+	"View": "Se",
+	"We\\'re lost in space. The page your were trying to view does not exist.": "Vi er tabt i rummet. Siden du forsøgte at se findes ikke.",
+	"Whoops": "Ups",
+	"Whoops!": "Ups!",
+	"With Trashed": "Med slettede",
+	"Write": "Skrive",
+	"could not be found.": "Kunne ikke blive fundet.",
+	"total": "totalt",
+	"January": "Januar",
+	"February": "Februar",
+	"March": "Marts",
+	"April": "April",
+	"May": "Maj",
+	"June": "Juni",
+	"July": "Juli",
+	"August": "August",
+	"September": "September",
+	"October": "Oktober",
+	"November": "November",
+	"December": "December",
+	"Afghanistan": "Afghanistan",
+	"Aland Islands": "Ålandsøerne",
+	"Albania": "Albanien",
+	"Algeria": "Algeriet",
+	"American Samoa": "Amerikansk Samoa",
+	"Andorra": "Andorra",
+	"Angola": "Angola",
+	"Anguilla": "Anguilla",
+	"Antarctica": "Antarktis",
+	"Antigua And Barbuda": "Antigua og Barbuda",
+	"Argentina": "Argentina",
+	"Armenia": "Armenien",
+	"Aruba": "Aruba",
+	"Australia": "Australien",
+	"Austria": "Østrig",
+	"Azerbaijan": "Aserbajdsjan",
+	"Bahamas": "Bahamas",
+	"Bahrain": "Bahrain",
+	"Bangladesh": "Bangladesh",
+	"Barbados": "Barbados",
+	"Belarus": "Hviderusland",
+	"Belgium": "Belgien",
+	"Belize": "Belize",
+	"Benin": "Benin",
+	"Bermuda": "Bermuda",
+	"Bhutan": "Bhutan",
+	"Bolivia": "Bolivia",
+	"Bosnia And Herzegovina": "Bosnien-Hercegovina",
+	"Botswana": "Botswana",
+	"Bouvet Island": "Bouvetøen",
+	"Brazil": "Brasilien",
+	"British Indian Ocean Territory": "British Indian Ocean Territory", //TODO: missing
```

Comment:
JSON is invalid because of this line. Please, fix it.

Reply:
Oh it does not support Æ, Ø, Å, my bad, thought those symbols were supported.